src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebWorkflowRouteConfig.kt

```diff
@@ -3,19 +3,23 @@ package org.vaccineimpact.orderlyweb.app_start.routing.web
 import org.vaccineimpact.orderlyweb.*
 import org.vaccineimpact.orderlyweb.app_start.RouteConfig
 import org.vaccineimpact.orderlyweb.controllers.web.WorkflowRunController
+import spark.route.HttpMethod
 
 object WebWorkflowRouteConfig : RouteConfig
 {
     private val runReports = setOf("*/reports.run")
 
     override val endpoints: List<EndpointDefinition> = listOf(
-            WebEndpoint("/workflows/:key", WorkflowRunController::class, "getWorkflowRunDetails")
-                    .json()
-                    .secure(runReports)
-                    .transform(),
-            WebEndpoint("/workflows", WorkflowRunController::class, "getWorkflowRunSummaries")
-                    .json()
-                    .secure(runReports)
-                    .transform()
+        WebEndpoint("/workflow", WorkflowRunController::class, "createWorkflowRun", HttpMethod.post)
```

Comment:
Have added this endpoint to `web` rather than `api`. Should eventually end up in the latter but would like to stabilise before publishing beyond UI.